XML/TestCases/BenchmarkTests-MPI.xml

```diff
@@ -0,0 +1,35 @@
+<TestCases>
+<!-- MVAPICH MPI -->
+    <test>
+		<testName>INFINIBAND-MVAPICH-MPI-OMB-2VM</testName>
+		<testScript>VERIFY-INFINIBAND-MultiVM.ps1</testScript>
+		<setupType>TwoVM1Dep</setupType>
+		<OverrideVMSize>Standard_HC44rs</OverrideVMSize>
+		<files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\TestRDMA_MultiVM.sh,.\Testscripts\Linux\SetupRDMA.sh</files>
+		<Priority>3</Priority>
+		<TestParameters>
+			<param>install_ofed="INFINIBAND_INSTALL_OFED"</param>
+			<param>install_ofed_from_extension="INFINIBAND_INSTALL_OFED_FROM_EXTENSION"</param>
+			<param>num_reboot="INFINIBAND_MVAPICH_TOTAL_REBOOT_COUNT"</param>
+			<param>mpi_settings="INFINIBAND_OMB_MVAPICH_MPI_SETTINGS"</param>
+			<param>ib_nic="INFINIBAND_IB_NIC"</param>
+			<param>imb_mpi1_tests="INFINIBAND_MVAPICH_MPI_IMB-TESTNAMES"</param>
+			<param>mpi1_ppn=INFINIBAND_MVAPICH_PPN</param>
+			<param>imb_mpi1_tests_iterations=INFINIBAND_MPI_TEST_COUNT</param>
+            <param>omb_p2p_tests="INFINIBAND_MVAPICH_OMB_P2P_TESTNAMES"</param>
```

Comment:
Please check out indentation for the next 4 lines.

Reply:
done in revision. strangely the indentation shows correctly in vs code and not in commit sometimes.